### PR TITLE
[EOC-546] add animations to add member form - PART ONE

### PR DESCRIPTION
**Pull request by adamklepacz**
This is part ONE of this task. The second part will be added in a separated pull request.
It includes: 
- clear button, 
- click outside
- email validation 

Zrzut ekranu 2019-10-18 o 09 27 52
Zrzut ekranu 2019-10-18 o 09 27 31
Zrzut ekranu 2019-10-18 o 09 27 25
Zrzut ekranu 2019-10-18 o 09 27 08

### CODE REVIEW

File: client/src/locales/en.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
   "common.button.confirm": "Confirm",
   "common.button.dashboard": "Go to dashboard",
   "common.button.reset-filter": "Reset filter",
+  "common.button.reset-input": "Reset input",
```
Comment (marekrozmus) [inline client/src/locales/en.json]: just 'Clear'

File: client/src/common/components/Members/components/MembersForm.jsx [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
 
 import Preloader, {
   PreloaderSize,
   PreloaderTheme
 } from 'common/components/Preloader';
 import { KeyCodes } from 'common/constants/enums';
 import { IntlPropType } from 'common/constants/propTypes';
+import { CloseIcon } from 'assets/images/icons';
 import './MembersForm.scss';
+import { validateWith } from 'common/utils/helpers';
```
Comment (alemikolo) [inline client/src/common/components/Members/components/MembersForm.jsx]: styles at the bottom of imports?

File: client/src/common/components/Members/components/MembersForm.jsx [hunk 3]
```diff
@@ -65,48 +86,86 @@ class MembersForm extends PureComponent {
 
   handleBlur = () => this.setState({ isFocused: false });
 
+  handleClickOutside = event => {
```
Comment (alemikolo) [inline client/src/common/components/Members/components/MembersForm.jsx]: What happens when you click outside during sending invitations? Does "cannot setState on unmounted component..." error appears or something?

Comment (adamklepacz) [reply client/src/common/components/Members/components/MembersForm.jsx]: Everything works as expected. No errors

**Pull request merged**